packages/docs/content/docs/developer-updates/2025-12-19-weekly-update-15.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Weekly Update #15
+---
+
+# Weekly Update #15
+
+### TL;DR
+
+This week includes several improvements and bug fixes across the VSCode extension and Web UI. 
+
+A mix of workflow enhancements, UI polish, and reliability fixes landed. Here are the highlights:
+
+
+### ✨ Enhancements
+
+- **Improved Worktree Selector in VSCode Web UI:** Added support for creating a new worktree directly from the dropdown, with a clearer “New Worktree” option and updated icon. Also includes tooltips and improved labeling for a smoother workflow. **[#896](https://github.com/TabbyML/pochi/issues/896)**
+
+
+- **Open diff directly from a task:** Clicking a task with diffs now opens both the task and its diff panel, making it easier to review changes directly from the task list. **[#849](https://github.com/TabbyML/pochi/issues/849)**
+
+- **Quick task creation from the Worktree sidebar:** Added a `+` button next to each worktree to create a new task. **[#877](https://github.com/TabbyML/pochi/issues/877)**
+
+
+- **New keyboard shortcut:** Added a `Ctrl+` keybinding in VSCode to instantly toggle the Pochi layout or open the terminal, improving keyboard-first navigation. **[#881](https://github.com/TabbyML/pochi/issues/881)**
```

Comment:
should we mention that this keybinding need to be enabled in pochi advanced settings?
```json
"pochi.advanced": {
    "enablePochiLayoutKeybinding": true,
}
```